### PR TITLE
extra_features/influxdb_adapter - dockerfiles for the exporter, grafana, influxdb

### PR DESCRIPTION
**Pull request by kmoppel**
Together with Senza templates that one can optionally use for AWS deploys

### CODE REVIEW

File: extra_features/influxdb_adapter/dockerfiles/grafana/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#ENV GRAFANA_PKG=grafana-latest.linux-x64.tar.gz
+ENV GRAFANA_PKG=grafana-2.1.3.linux-x64.tar.gz
+
+ENV GRAFANA_URL=https://grafanarel.s3.amazonaws.com/builds/${GRAFANA_PKG}
+
+RUN apt-get update && apt-get install -y wget
+RUN wget -q ${GRAFANA_URL}
+
+RUN mkdir grafana && tar xf ${GRAFANA_PKG} -C grafana --strip-components 1
+RUN chmod -R 777 grafana
```
Comment (a1exsh) [inline extra_features/influxdb_adapter/dockerfiles/grafana/Dockerfile]: Doesn't strike me as a really good idea.

File: extra_features/influxdb_adapter/dockerfiles/influxdb/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+EXPOSE 8083
+EXPOSE 8086
+EXPOSE 8088
+
+# change config to disable writing of self-monitoring metrics
+RUN sed -i 's/store-enabled = true/store-enabled = false/g' /etc/opt/influxdb/influxdb.conf
+RUN mkdir /var/run/influxdb && chmod -R 777 /var/run/influxdb && chmod -R 777 /var/opt/influxdb
+
```
Comment (a1exsh) [inline extra_features/influxdb_adapter/dockerfiles/influxdb/Dockerfile]: Same thing again: `chmod -R 777`?

Comment (kmoppel) [inline extra_features/influxdb_adapter/dockerfiles/influxdb/Dockerfile]: Yes, I see your point. But well, it's all still within the docker image, so I see no real threat. I couldn't run without it, as I'm changing the default config files dynamically.

Comment (hjacobs) [inline extra_features/influxdb_adapter/dockerfiles/influxdb/Dockerfile]: chmod 777 is definitely better (from security point of view) than running the Docker container as "root"

**Comment by kmoppel** [comment]
was pushed to master (https://github.com/zalando/PGObserver/commit/55f79da3f8f63bcb15625e975222f6dae88378ba) from a diff, branch got too messy for a good overview

**Pull request closed without merging**